components/Buttons/tests/unit/ButtonsTests.m

```diff
@@ -423,4 +532,23 @@ - (void)testAdjustsFontProperty {
                              @"Font size should be equal to MDCFontTextStyleButton's.");
 }
 
+#pragma mark utilities
+
+- (NSString*)controlState:(UIControlState)controlState {
+  NSMutableString *string = [NSMutableString string];
+  if (UIControlStateNormal) {
```

Comment:
lets copy the function from the mutator tests:
https://github.com/material-components/material-components-ios/blob/develop/components/Buttons/tests/unit/ButtonTitleColorAccessibilityMutatorTests.m#L116